### PR TITLE
Create MR when new build-definitions pipelines are created

### PR DESCRIPTION
**Pull request by Michkov**
Clone infra-deployments, update the code, push and create MR.

Testing MR produced by the pipeline: #149 
Secret is set on the stage instance.

### CODE REVIEW

File: components/build/.tekton/trigger-template.yaml [hunk 1]
```diff
@@ -63,14 +73,53 @@ spec:
...
+                - name: BASE_IMAGE
+                  value: registry.redhat.io/openshift-pipelines/pipelines-git-init-rhel8@sha256:af7dd5b3b1598a980f17d5f5d3d8a4b11ab4f5184677f7f17ad302baa36bd3c1
+                - name: GIT_USER_NAME
+                  value: Build deployer
+                - name: GIT_SCRIPT
+                  value: |
+                    ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
+                    git clone https://github.com/redhat-appstudio/infra-deployments
+                    cd infra-deployments
+                    git remote add push git@github.com:Michkov/infra-deployments.git
```
Comment (mmorhun) [inline components/build/.tekton/trigger-template.yaml:89]: We shouldn't use personal accounts for such tasks

File: components/build/.tekton/trigger-template.yaml [hunk 2]
```diff
@@ -63,14 +73,53 @@ spec:
...
+                    done
+                    git commit -a -m "Build definitions update" -m "from $(params.git-url) $(params.revision)"
+                    git push -f --set-upstream push build-definitions-update
+
+                    set +x
+                    curl -H "Authorization: token $(cat /root/.ssh/token)" \
+                      -X POST \
+                      -H "Accept: application/vnd.github.v3+json" \
+                      https://api.github.com/repos/redhat-appstudio/infra-deployments/pulls \
+                      -d '{"head":"Michkov:build-definitions-update","base":"main","title":"Build definitions update"}'
```
Comment (mmorhun) [inline components/build/.tekton/trigger-template.yaml:103]: Same as above

**Review by mmorhun — CHANGES_REQUESTED** [review]

File: components/build/.tekton/trigger-template.yaml (hunk 1 above)
Comment (Michkov) [reply components/build/.tekton/trigger-template.yaml]: Updated to new org redhat-appstudio-mr-creation

File: components/build/.tekton/trigger-template.yaml (hunk 2 above)
Comment (Michkov) [reply components/build/.tekton/trigger-template.yaml]: Updated to new org redhat-appstudio-mr-creation

**Review by mmorhun — APPROVED** [review]
LGTM

**Comment by mmorhun** [comment]
Also it would be good to grant access to the MR org for, at least, leads and architects.

**Pull request merged**